### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by tafodinho**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    margin-bottom: 25px
+}
+.align-center{
+    text-align: center;
+}
+.inline{
+    display: inline;
+}
+.top-section-margin{
+    margin: 5% 5px 5% 5px;
```
Comment (zlayabekrija) [inline assets/css/style.css]: This could be written as margin: 5% 5px 
But please don't use % for padding or margin.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Nice work, yet bootstrap isn't allowed for this project. I've seen that you have used a minimum of BS, but still I would like to ask you to remove any bootstrap from this project.
Kindly fix and resubmit review request.
Happy Coding

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+}
+top-section {
+    margin-bottom: 20px
+}
+/*inline styling*/
+.inline{
+    display: inline;
+}
+.inline-list {
```
Comment (oluwoleilesanmi) [inline assets/css/style.css:48]: It is good practice to have a reset.css file.
- reset.css helps with avoiding repetition.
- reset.css helps with setting elements to default. 

The below should be moved to reset.css file.
```
margin: 0px;
padding: 0px;
```

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+.form-bottom {
+    margin-top: 45px;
+}
+.form-bottom * {
+    font-style: italic;
+    font-size: 12px;
+    margin-top: -15px;
+}
+.form-top-text1 {
```
Comment (oluwoleilesanmi) [inline assets/css/style.css:112]: property should be moved to reset.css file.
```
margin: 0px; 
```

File: assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    justify-content: center;
+}
+.vertical-align-m{
+    vertical-align: middle;
+}
+/*End of align*/
+.signup-margin-right{
+    margin-right: 5px;
+}
+.signup-text {
```
Comment (oluwoleilesanmi) [inline assets/css/style.css:38]: The two properties should be moved to reset.css file.
```
margin: 0px;
padding: 0px;
```

File: assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+
+.top-section-margin{
+    margin: 5% 5px;
+}
+.clear-both {
+    clear: both;
+}
+
+.form-section {
```
Comment (oluwoleilesanmi) [inline assets/css/style.css]: Remove empty selector.

**Review by oluwoleilesanmi — CHANGES_REQUESTED** [review]
Good job @tafodinho @offorma  its good that, your commit messages start with verbs, good use of relative units of measure ems\% and a nice and clean html. **make changes to css below and you are good to go**. well done

This is not compulsory but here is a [Css Linter](http://csslint.net) it helps with keeping up with css best practices.

**Review by oluwoleilesanmi — APPROVED** [review]
@tafodinho @offorma :+1: job well done

**Pull request merged**